pages/property-filter/single-use-properties.page.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do I understand correctly that this demo currently does not work?

How about not adding this demo into the mainline for now?

Reply:
Alright.